datapublic/common_init.py

```diff
@@ -36,12 +35,6 @@ def configure_logging(command: Optional[str] = None):
         wrapper_class=structlog.stdlib.BoundLogger,
         processors=[
             structlog.stdlib.add_log_level,  # required before SentryProcessor()
-            # sentry_sdk creates events for level >= ERROR. Getting breadcrumbs from structlog isn't supported
-            # without a lot of custom work. See https://github.com/kiwicom/structlog-sentry/issues/25.
-            # The SentryJsonProcessor is used to protect against event duplication.
-            # It adds loggers to a sentry_sdk ignore list, making sure that the message logged is
-            # not reported in addition to the exception stack trace.
-            SentryJsonProcessor(level=logging.ERROR, tag_keys="__all__"),
```

Comment:
`SentryJsonProcessor` was [removed in version 2.0.0](https://github.com/kiwicom/structlog-sentry/releases/tag/v2.0.0). IIRC, we don't track sentry alerts anymore, so I've opted just to remove these configuration options. 

If we want to preserve this, I can either pin the versions to `1.x` or work to refactor this in version `2.0`